new-website/src/components/location/edit-page.jsx

```diff
@@ -11,8 +12,15 @@ function fetchDoc(docId) {
 }
 
 async function createDoc(partnerIds) {
-  if (!partnerIds || partnerIds.length === 0)
+  if (!partnerIds || partnerIds.length === 0) {
+    Sentry.withScope(scope => {
+      scope.setLevel("warning")
+      Sentry.captureEvent({
+        message: "Attempt to create map entry without associated partner ID"
+      })
+    })
```

Comment:
@c0un7-z3r0 When testing locally, I don't see anything popping up in our Sentry dashboard. Am I doing something wrong, or does it only work once it's deployed? (or does it have a lag?)

Reply:
It only works on deployed versions :D We do not want to track local dev errors ;)